docs/vercel/public/sakura.css

```diff
@@ -99,6 +99,12 @@ a:hover {
   border-bottom: 2px solid #222222;
 }
 
+h3 a,
+h3 a:hover,
+h3 a:visited {
+  color: #222222;
+}
+
```

Comment:
I think any custom CSS should go in a separate file, so that there won't be any conflict to resolve if we decide to update `sakura.css`. Also, maybe this should be a class instead of set on the HTML tags, so we won't have to update the CSS if we ever change the HTML element (e.g. `h3` -> `h4`).

Reply:
https://github.com/o2sh/onefetch/pull/922/commits/8b0dd3b25d8418150667b918ea1e1a0e78269ef6